### PR TITLE
Add test for inserting multiple rows provided by #5158

### PR DESCRIPTION
**Pull request by mmuzikar**
//`@db-insert-multiple-rows`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 1]
```diff
@@ -300,3 +300,55 @@ Feature: Integration - DB to DB
 
     Then validate that number of all todos with task "Joe" is "0"
     Then validate that number of all todos with task "Jimmy" is "1"
+
+  @db-insert-multiple-rows
+  Scenario: Inserting multiple rows 
+    Given Set Todo app credentials
+    Then inserts into "todo" table
+      | Joe |
+    Then inserts into "todo" table
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: `And`

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 2]
```diff
@@ -300,3 +300,55 @@ Feature: Integration - DB to DB
...
+    And create mapping from "body.task" to "task"
+
+    And click on the "Done" button
+    And publish integration
+    And set integration name "DB Insert multiple rows"
+    And publish integration
+    Then wait until integration "DB Insert multiple rows" gets into "Running" state
+
+    Then check that query "SELECT * FROM contact WHERE first_name = 'Jimmy'" has some output
+    Then check that query "SELECT * FROM contact WHERE first_name = 'Joe'" has some output
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: `And`

File: ui-tests/src/test/resources/features/integrations/db-to-db.feature [hunk 3]
```diff
@@ -300,3 +300,55 @@ Feature: Integration - DB to DB
...
+        | security | authType      | HTTP Basic Authentication     |
+        | details  | connectorName | TODO-API                      |
+        | details  | host          | http://todo.syndesis.svc:8080 |
+        | details  | baseUrl       | /api                          |
+    And navigate to the "Connections" page
+    And created connections
+        | TODO-API | todo | TODO-app | no validation |
+
+    When navigate to the "Home" page
+    And click on the "Create Integration" button
```
Comment (avano) [inline ui-tests/src/test/resources/features/integrations/db-to-db.feature]: it seems there are some whitespaces at the end of the line, please check that :-)

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Integration - Database | Smoke - Periodic invocation to Insert 
Connection - CRUD | Validate all credentials 
```

**Pull request merged**